blog/posts/announcing-melange-3.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Announcing Melange 3'
+date: 2024-02-13
+author: Antonio Monteiro
+gravatar: 45c2052f50f561b9dc2cae59c777aecd794f57269fa317f9c9c3365c2e00d16f
+twitter: '@_anmonteiro'
+---
+
+We are excited to announce the release of [Melange
+3](https://github.com/melange-re/melange/releases/tag/3.0.0-51), the latest
+version of our backend for the OCaml compiler that emits JavaScript.
+
+This new version comes packed with significant changes, improvements, and a few
+necessary removals to ensure a more streamlined and efficient experience for our
+users. This new version is both leaner and more robust. We focused on fixing
+crashes and removing obsolete functionality, improving the developer and
+troubleshooting experience, increasing OCaml compatibility and JavaScript FFI
+integration.
+
+---
+
+Here's a rundown of the key updates in Melange 3. Check the [Melange
+documentation](https://melange.re/v3.0.0/) for further resources.
+
+## Major Changes and Removals
+
+In Melange 3, `Belt` is no longer a dependency for the Melange `Stdlib`.
+Libraries that depend on the Belt modules will need to include `(libraries
+melange.belt)` in their build configuration.
+
+The `@bs` / `@bs.*` attributes have been replaced. Users of Melange should now
+utilize `[@u]` for uncurried application and `[@mel.*]` for FFI attributes.
+Additionally:
+
+- `[@mel.val]` has been removed as it was redundant in the Melange FFI​​.
+- `[@mel.splice]` was removed in favor of `[@mel.variadic]`
+
+For this release, most modules in the `Js` namespace had their APIs unified,
+deduplicated and refactored. In cases such as `Js.Int`, `Js.Date`, `Js.Re`,
+`Js.Float`, `Js.String`, some functions were changed from pipe-first to
+pipe-last and labeled arguments were added; and incorporating those made others
+obsolete, which we removed. Modules such as `Js.List`, `Js.Null_undefined`,
+`Js.Option`, `Js.Result` and `Js.Cast` are also no longer present in Melange 3.
+Alternatives within `Stdlib` or `Belt` are instead​​ recommended.
+
+## New Features and Enhancements
```

Comment:
Should we mention that Melange is now releasing versions for 4.14 and 5.1? That is quite a big deal, imo.

Reply:
great catch! this is definitely something we should mention